contrib/guix/libexec/build.sh

```diff
@@ -245,7 +245,7 @@ CONFIGFLAGS="--enable-reduce-exports --disable-bench --disable-gui-tests --disab
 # CFLAGS
 HOST_CFLAGS="-O2 -g"
 case "$HOST" in
-    *linux*)  HOST_CFLAGS+=" -ffile-prefix-map=${PWD}=." ;;
+    *linux*)  HOST_CFLAGS+=" -ffile-prefix-map=${PWD}=. -fPIC" ;;
```

Comment:
Could you remove `-fPIC`? I'd like to keep these hacks separate from branding because they relate to ongoing issues that could be looked into.